src/main/java/uk/gov/hmcts/divorce/common/service/ApplyForFinalOrderService.java

```diff
@@ -37,4 +45,19 @@ public CaseDetails<CaseData, State> applyForFinalOrderAsApplicant2(final CaseDet
             progressFinalOrderState
         ).run(caseDetails);
     }
+
+    public static List<String> validateApplyForFinalOrder(final CaseData caseData, boolean isApplicant2Event) {
```

Comment:
Does this need to be a static method on a Service?

Reply:
Updated